README.md

```diff
@@ -6,8 +6,6 @@ To quickly learn how to run cleanlab on your own data, first check out the [quic
 
 ## Table of Contents
 
-|     | Example                                                                                        | Description                                                                                                                                                                                                                                                                  |
-| --- | ---------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
This header was missing for some reason, so the Table of contents was broken.  Before requesting PR reviews, please do always look at your PRs in the GH viewer under `Files Changed` to see what the delta is, and also imagine you are the Reviewer for ~10min.

Reply:
Ack, sorry!